File: src/main/scala/org/cloudio/morpheus/mail/morpheus/MailOwner.scala
package org.cloudio.morpheus.mail.morpheus

import java.util.Date

import org.morpheus.dimension

/**
 * Created by zslajchrt on 24/08/15.
 */
trait MailOwner {
  def nick: String

  def firstName: String

  def lastName: String

  def email: String

  def isMale: Boolean

  def birthDate: Date
}

File: src/main/java/org/cloudio/morpheus/mail/MailOwner.java
package org.cloudio.morpheus.mail;

import java.util.Date;

/**
 * Created by zslajchrt on 24/08/15.
 */
public interface MailOwner {
    String nick();

    String firstName();

    String lastName();

    String email();

    boolean isMale();

    Date birthDate();
}

